File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int left = 0, right = 1;
        int n = nums.size();
        int ans = 1;
        while(right < n){
            if(nums[left] == nums[right]){
                right++;
            }
            else{
                nums[++left] = nums[right++];
                ans++; 
            }
        }

        return ans;
    }
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        lastVal = nums[0]
        n = len(nums)
        k = 1
        for i in range(n):
            if i == 0:
                continue
            if nums[i] != lastVal:
                nums[k] = nums[i]
                lastVal = nums[k]
                k += 1
        
        return k
        